pkg/web_app/lib/src/screenshot_carousel.dart

```diff
@@ -57,6 +57,7 @@ void _setEventForScreenshot() {
   for (final thumbnail in thumbnails) {
     thumbnail.parent!.onClick.listen((event) {
       showElement(carousel);
+      document.body!.style.overflow = 'hidden';
```

Comment:
At one point we were planning to restrict such inlined styles with Content Security Policy headers. Let's use class names instead, e.g.
```
.overflow-hidden { overflow: hidden;}
.overflow-auto {overflow: auto;}
```
and
```
document.body!.classes.remove('overflow-auto');
document.body!.classes.add('overflow-hidden');
`````